style/palette.ts

```diff
@@ -44,13 +44,29 @@ const colors: object = {
   black: '#000',
 };
 
-const fontSizes: object = {}; // 하람
+const fontSizes: object = {
+  '10': '10px',
+  '12': '12px',
+  '14': '14px',
+  '16': '16px',
+  '18': '18px',
+  '20': '20px',
+  '22': '22px',
+  '24': '24px',
+  '28': '28px',
+  '36': '36px',
+  '44': '44px',
```

Comment:
```suggestion
  '44': css`
    font-size: 44px
  `
```
로 작성해야 바로 가져다 쓸 수 있을 것 같아요